libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -1131,7 +1131,7 @@ export function updateParameterWithValues(
                     isUnknown: true,
                   };
 
-                  parameters.push(transformInputParameter(restInputParameter, propertyValue, /* invisible */ false));
+                  parameters.push(transformInputParameter(restInputParameter, propertyValue, /* invisible */ true));
```

Comment:
From reading the code I'm assuming `invisible` should true since `createInvisibleParameter` is true

Reply:
Please dont update this unless this is fixing some scenario.

Reply:
Sure I didn't know of any existing scenarios to test it with, but the code reads like it's doing the wrong thing.

Undid the change